draft-ietf-taps-interface.md

```diff
@@ -655,9 +655,13 @@ Selection, and Connection Properties, as well as defaults for Message Properties
 They are collected into a TransportProperties object to be passed into a Preconnection object:
 
 ~~~
-TransportProperties := NewTransportProperties()
+TransportProperties := NewTransportProperties(profile?)
```

Comment:
I think this is very useful in the appendix, but adding language here in the normative section might lead to some of the confusion that putting this in the appendixi is meant to avoid. Consider leaving this section unchanged, and make the modification to NewTransportProperties() in the appendix where properties are introduced.

Reply:
Hmm… I think it is a much worse idea to modify function signatures in the appendix